### DISCUSSION TITLE
你好，我自定义一个surface 进行离屏渲染，如何指定帧缓冲区？

### DISCUSSION THREAD

**Discussion by gzzhangbl** [Tencent/tgfx#200] [Q&A] [answered]
你好，我自定义一个surface 进行离屏渲染，如何指定帧缓冲区？

**Comment by domchen** [comment, marked as answer]
自己的framebuffer包装成BackendRenderTarget， 然后用Surface::MakeFrom(BackendRenderTarget)创建Surface。
